.travis.yml

```diff
@@ -1,8 +1,8 @@
 language: ruby
 cache: bundler
 bundler_args: '--without production development'
-rvm:
-  - 2.3.0
+#rvm:
+  #- 2.3.0
```

Comment:
Would you mind to just remove them completely instead of commenting the lines out? Thanks!